examples/go-code-tutorial/BUILD.bazel

```diff
@@ -0,0 +1,30 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library")
```

Comment:
nit: it seems a bit surprising to me that "tutorial" is in the name of this folder, I'd expect a lot of users will just look in rules_go and be happy to find an example, but not know or care that there's also a tutorial accompanying it.

Reply:
I suck at naming 😆 - Do you have a better one?

Reply:
how about `examples/go-code` simply drop tutorial. Makes it clear and the tutorial cna rference the already extant example.

Reply:
We are in rules_go, so I would assume it is go ... yah.  Do we have other ideas?

Reply:
I changed the name to bazel-gazelle.